.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,59 @@
+name: build-and-test
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
```

Comment:
Any specific reason you chose multiple jobs over one job with multiple steps?

Reply:
Yeah.. Build and test jobs can execute in parallel. Verification will take less time.